docs/gettingstarted.rst

```diff
@@ -229,8 +229,11 @@ Deploy on CERN infrastructure
 
    .. code-block:: console
 
-      (reana) $ reana-cluster -f reana-cluster-CERN.yaml --cephfs init
+      (reana) $ reana-cluster -f reana-cluster-CERN.yaml --cephfs --ui init
```

Comment:
`cli/cluster.py` `init()` does not have `--ui` flag. Wouldn't it be error-prone?
(thinking of `reana-cluster --ui init` vs `reana-cluster init --ui`)